README.md

```diff
@@ -103,6 +103,7 @@ Ziggurat currently uses rust's standard test runner, a simple `cargo test -- --t
 Logs are disabled by default, as they usually just add noise and slow down the test. They can be very useful for debugging and can be enabled on a test case level.
 
 Ziggurat's `SyntheticNode` supports `tracing` - this can be enabled by inserting a call to `synthetic_node::enable_tracing()` inside the test case.
+Use `--nocapture` option combined with the `RUST_LOG` environment variable to show logs in the `stdout`. Configure the `RUST_LOG` environment variable to select the [logging level](https://docs.rs/env_logger/latest/env_logger/#enabling-logging). For example: `RUST_LOG=trace cargo test -- --test-threads=1 --nocapture`.
```

Comment:
```suggestion
Use the `--nocapture` option combined with the `RUST_LOG` environment variable to show logs from `stdout`. Configure the `RUST_LOG` environment variable to select the [logging level](https://docs.rs/env_logger/latest/env_logger/#enabling-logging). For example: `RUST_LOG=trace cargo test -- --test-threads=1 --nocapture`.
```

Reply:
Should I commit the suggestion, and then later we can squash it before merging it on the main branch?